client/src/components/Overview/AddToCart.jsx

```diff
@@ -30,16 +30,15 @@ function AddToCart(props) {
       return;
     }
 
-    const quantity = Math.min(styles[styleSelector].skus[sizeSelection].quantity, 15);
-
+    const quantity = styles[styleSelector].skus[sizeSelection] && Math.min(styles[styleSelector].skus[sizeSelection].quantity, 15);
     setQuantityOptions(
       _.range(quantity + 1).map((amount) => (
         <option key={amount} value={amount}>{amount}</option>
       ))
     );
   }, [sizeSelection]);
 let buttonNoButton;
-var alertMe;
+var alertMe
```

Comment:
Can you use ES6 `let` or `const` to keep consistency with other files in the project?

And add the missing `;`